packages/design-system-tailwind-preset/tsconfig.build.json

```diff
@@ -3,7 +3,8 @@
   "compilerOptions": {
     "baseUrl": "./",
     "outDir": "./dist",
-    "rootDir": "./src"
+    "rootDir": "./src",
+    "types": ["node"]
```

Comment:
Why is this necesary?

Reply:
See https://github.com/MetaMask/metamask-design-system/pull/60/files#r1811558493